machine-learning/app/config.py

```diff
@@ -81,6 +83,7 @@ def __init__(self) -> None:
             console=console,
             rich_tracebacks=True,
             tracebacks_suppress=[*self.excluded, concurrent.futures],
+            tracebacks_show_locals=LOG_LEVEL == logging.DEBUG,
```

Comment:
Is this change intentional or only for debugging?

Reply:
Intentional since it makes debugging this sort of thing easier